client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
```

Comment:
The response from the server, actually, can be either `string`, or `false`, so thanks for catching this.
![screen shot 2017-05-26 at 12 10 42](https://cloud.githubusercontent.com/assets/233601/26500455/68675b70-420c-11e7-8d2d-22051436a263.png)
And it depends on the privacy settings of the user.

Don't struggle too much with this. The response from the server goes straight to state, and has many fields that are never shown and should be ignored. In fact, on _Post_ and the _Reader_, the user's email is never shown. We need to do a cleanup of what is saved to state.
For the new management interface, if we have to show the email, a `truthy` check should suffice.